### PR TITLE
Better: Update SurveySdk documentation (realtime responses import, additional URL parameters). RD-21384

### PR DESCRIPTION
**Pull request by DrissTM**
Hey!

**Ticket(s)**:
- [RD-21384](https://jira.ringcentral.com/browse/RD-21384)

**Change(s)**:
- Add documentation about realtime responses import for SurveySdk
- Add documentation about additional URL parameters for SurveySdk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Go check [specs/README.md](specs/README.md)
 
 ### Testing OpenAPI Specs
 
-This repo holds OpenAPI specs for Engage Digital. 
+This repo holds OpenAPI specs for Engage Digital.
```
[the remark this answers is not in the document]
Comment (aymebou) [reply README.md:32]: Thanks for fixing it 👍

File: docs/sdks/survey-sdk/responses-polling.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Here's an example of a valid response to the responses polling:
             {
               "value": "Better follow-up"
             }
-        ]
+          ]
```
[the remark this answers is not in the document]
Comment (aymebou) [reply docs/sdks/survey-sdk/responses-polling.md:36]: Thanks for fixing it 👍

File: docs/sdks/survey-sdk/quick-start.md [hunk 3]
```diff
@@ -22,6 +22,28 @@ Enter the follow required fields:
...
 | **Secret key** | Secret key that will be sent in a `X-SDK-SECRET` header by Engage Digital. |
 | **Verify secret key** | If this option is enabled Engage Digital will make sure that the bridge response contains the `X-SDK-SECRET` header and that its value is matching the one configured in Engage Digital. |
+| **Enable import of survey results via polling** | When this option is enabled Engage Digital will regularly (every 15 minutes) make HTTP requests to fetch responses, see [Responses Polling](../responses-polling). |
+| **Enable realtime import of survey results** | Enabling this option will display the survey **Realtime endpoint URL** which you will be able to use to push new responses to Engage Digital in realtime, see [Realtime Responses Import](../realtime-responses-import). |
+| **Additional URL parameters** | List of conversation-related data that can be added to the survey link that will be sent to the end-user, see [Additional URL parameters](#additional-url-parameters). |
 
 !!! warning
     To be able to save the survey your bridge must **at least** implement the survey configuration part (see [Configuration fetching](../configuration-fetching)).
```
Comment (aymebou) [inline docs/sdks/survey-sdk/quick-start.md:30]: Oh I see some doc update I'll be doing in the near future

File: docs/sdks/survey-sdk/realtime-responses-import.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
+# Realtime Responses Import
+
+As explained in [Getting Started](../quick-start) you can enable the import of responses in realtime, by doing so you'll obtain a **Realtime endpoint URL** which you'll be able to use to perform HTTP POST requests in order to import survey responses in realtime (see [Payload example](#payload-example) for more detail on the payload expected by Engage Digital).
```
Comment (aymebou) [inline docs/sdks/survey-sdk/realtime-responses-import.md:3]: > You'll obtain a **Realtime endpoint URL**

You don't explain where they'll obtain it, I find this a bit confusing WDYT ?

**Review by aymebou — CHANGES_REQUESTED** [review]

File: docs/sdks/survey-sdk/quick-start.md (hunk 3 above)
Comment (DrissTM) [reply docs/sdks/survey-sdk/quick-start.md:30]: hehe yes there's some things that will need to be updated, I'll create a ticket for that

File: docs/sdks/survey-sdk/realtime-responses-import.md (hunk 4 above)
Comment (DrissTM) [reply docs/sdks/survey-sdk/realtime-responses-import.md]: I rephrased this whole part in [a1fcef2](https://github.com/ringcentral/engage-digital-api-docs/pull/80/commits/a1fcef26c49b65d7cd05340e638c1331a1221772), let me know if it's clearer this way

Comment (aymebou) [reply docs/sdks/survey-sdk/realtime-responses-import.md]: Perfect ! Even I wasn't sure it was in the survey config page either 👍

**Comment by aymebou** [comment]
LGTM

**Pull request merged**